bundles/org.openhab.core.persistence/src/main/java/org/openhab/core/persistence/extensions/PersistenceExtensions.java

```diff
@@ -67,52 +91,193 @@ public PersistenceExtensions(@Reference PersistenceServiceRegistry registry) {
      * @param serviceId the name of the {@link PersistenceService} to use
      */
     public static void persist(Item item, String serviceId) {
+        internalPersist(item, serviceId);
+    }
+
+    private static void internalPersist(Item item) {
+        String serviceId = getDefaultServiceId();
+        if (serviceId == null) {
```

Comment:
Shouldn't this result in a warning similar to the one on l. 111? ("No default persistence service configured")

Reply:
`getDefaultServiceId()` already logs a warning.